packages/docusaurus/src/commands/swizzle.ts

```diff
@@ -123,8 +123,8 @@ function colorCode(
   );
 
   return [
-    ...greenComponents.map((component) => chalk.green(component)),
-    ...redComponents.map((component) => chalk.red(component)),
+    ...greenComponents.map((component) => chalk.green(`safe: ${component}`)),
+    ...redComponents.map((component) => chalk.red(`unsafe: ${component}`)),
```

Comment:
can we also update the legend at the top to use "safe" / "unsafe" wording too?

Also please include a screenshot so that we can review this without running the code easily